src/components/Button/interface.tsx

```diff
@@ -0,0 +1,5 @@
+export interface ButtonProps {
```

Comment:
Could we use in this case` type PropsOptional`?

Reply:
We don't need any of these props to be required elsewhere and having them optional is the only version of interface we need in our app - that's why we don't need to create another type with using PropsOptional.